shared/middleware/helpers/storage_apps.rb

```diff
@@ -14,7 +14,7 @@ def initialize(storage_id)
   def create(value, ip:, type: nil, published_at: nil)
     timestamp = DateTime.now
     row = {
-      storage_id: @storage_id,
+      storage_id: @storage_id.to_i,
```

Comment:
Looks like it is coming from [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/test_helper.rb#L449-L451).